src/vs/workbench/browser/parts/statusbar/media/statusbarpart.css

```diff
@@ -47,16 +47,23 @@
 	vertical-align: top;
 	max-width: 40vw;
 }
-
 .monaco-workbench .part.statusbar > .items-container > .statusbar-item.has-beak {
 	position: relative;
 }
 
-.monaco-workbench .part.statusbar > .items-container > .statusbar-item.has-beak:before {
+.monaco-workbench .part.statusbar > .items-container > .statusbar-item > .beak-div-show {
```

Comment:
Rather than having a `beak-div-show`, why not add a CSS class name to the beak container (`status-item-beak-container`) and then use the existing `has-beak` rule to make this work, e.g.:

```
.monaco-workbench .part.statusbar > .items-container > .statusbar-item.has-beak > .status-bar-beak-container
```

Reply:
yes sure

Reply:
I removed a line-break (empty line) by mistake, should I commit again (with the line break)?

Reply:
Sure.